src/shared/components/plots/TablePlotUtils.ts

```diff
@@ -6,8 +6,8 @@ function getUniqueSampleKeyToCategories(
 ) {
     const ret: {
         [sampleKey: string]: {
-            horz?: string | (string[]);
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)